qt/applications/workbench/workbench/projectrecovery/test/test_projectrecovery.py

```diff
@@ -36,6 +36,12 @@ def setUp(self):
         self.multifileinterpreter = mock.MagicMock()
         self.pr = ProjectRecovery(self.multifileinterpreter)
         self.working_directory = tempfile.mkdtemp()
+        # Make sure there is actually a different modified time on the files by using sleeps
+        self.firstPath = tempfile.mkdtemp()
```

Comment:
Could we use `os` to set the modification times like so: https://nitratine.net/blog/post/change-file-modification-time-in-python/

Sleep can lead to flaky tests, for example if the underlying filesystem has a time resolution of 2s (e.g. FAT32 despite how unlikely)

Reply:
I've changed the code to use os.utime instead of sleep. I noticed the unit test also wasn't deleting the temp folders that are having their datestamp manipulated so I've added some lines to tearDown to achieve that